deploy/helm/charts/values.yaml

```diff
@@ -23,7 +23,7 @@ localpv:
     # For example : quay.io/ is a correct value here and quay.io is incorrect
     registry:
     repository: openebs/provisioner-localpv
-    tag: 2.5.0
+    tag: 2.6.0
```

Comment:
there is release.version above in the file needs to be updated as well

Reply:
Done.